glean/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 
 add_library(glean STATIC)
 
-if(NOT MSVC)
+if(NOT MSVC AND NOT XCODE)
```

Comment:
oh... why? XCODE means macos too.

Reply:
Yes, yes. Right now I just want to see if this works on CI, the code is not ready for review and yes, this will need to change  to iOS only.